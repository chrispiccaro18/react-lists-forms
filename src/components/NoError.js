import React from 'react';

export default function NoError() {
  return (
    <h1>All good here</h1>
  );
}
