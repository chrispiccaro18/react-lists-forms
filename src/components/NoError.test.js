import React from 'react';
import { shallow } from 'enzyme';
import NoError from './NoError';

describe('NoError component', () => {
  it('renders NoError component', () => {
    const wrapper = shallow(<NoError />);
    expect(wrapper).toMatchSnapshot();
  });
});
