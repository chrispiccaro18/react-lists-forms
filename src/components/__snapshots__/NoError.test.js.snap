// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoError component renders NoError component 1`] = `
<h1>
  All good here
</h1>
`;
